From: =?utf-8?q?Carl_F=C3=BCrstenberg?= <azatoth@gmail.com>
Date: Tue, 6 Jan 2015 23:43:30 +0100
Subject: use multiarch compatible plugin lib path

use OBS_PLUGIN_DESTINATION instead of hardcoded path for module_bin
---
 libobs/obs-nix.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libobs/obs-nix.c b/libobs/obs-nix.c
index f7f2af3..ea425d5 100644
--- a/libobs/obs-nix.c
+++ b/libobs/obs-nix.c
@@ -38,7 +38,7 @@ const char *get_module_extension(void)
 
 static const char *module_bin[] = {
 	"../../obs-plugins/" BIT_STRING,
-	OBS_INSTALL_PREFIX "lib/obs-plugins",
+	OBS_INSTALL_PREFIX "/" OBS_PLUGIN_DESTINATION
 };
 
 static const char *module_data[] = {
